packages/snippets/bin/snippets.dart

```diff
@@ -16,23 +19,47 @@ const String _kHelpOption = 'help';
 const String _kInputOption = 'input';
 const String _kLibraryOption = 'library';
 const String _kOutputOption = 'output';
+const String _kOutputDirectoryOption = 'output-directory';
 const String _kPackageOption = 'package';
 const String _kTemplateOption = 'template';
 const String _kTypeOption = 'type';
 
 const LocalFileSystem filesystem = LocalFileSystem();
 
-String getChannelName() {
+class GitStatusFailed implements Exception {
+  GitStatusFailed(this.gitResult);
+  final ProcessResult gitResult;
```

Comment:
nit: maybe add some blank lines before and after this line for seperation and to improve readability?

Reply:
OK, done.